pkg/install/hypershift.go

```diff
@@ -146,8 +146,8 @@ func (c *UpgradeController) RunHypershiftCleanup(ctx context.Context) error {
 }
 
 // This is run when:
-//   1) the controller starts up
-//   2) when the hypershift installation options (secrets and configmap) change
+//  1. the controller starts up
+//  2. when the hypershift installation options (secrets and configmap) change
```

Comment:
I don't know what auto updated these two lines. Not sure if it was a make command or something else. If it's not too much let's just change the comment.